docs/reference/contributing/target/sai.md

```diff
@@ -83,7 +83,7 @@ Hardware SAI/I2S capabilities.
 
 You can find the API and specification for the SAI API in the following header file:
 
-[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/development/feature-hal-spec-sai/group__hal__sai.html)
```

Comment:
the URL was correct, the generation was not.
If you agree i'll rebase to drop this commit.